packages/console/src/console-history/ConsoleHistoryItem.tsx

```diff
@@ -77,28 +79,30 @@ class ConsoleHistoryItem extends PureComponent<
 
       if (changes) {
         const { created, updated } = changes;
-        [...created, ...updated].forEach(object => {
-          hasButtons = true;
-          const { title } = object;
-          const key = `${title}`;
-          const btnDisabled =
-            disabled === undefined ||
-            disabled ||
-            (disabledObjects ?? []).indexOf(key) >= 0;
-          const element = (
-            <Button
-              key={key}
-              kind="primary"
-              onClick={() => this.handleObjectClick(object)}
-              className="btn-console-object"
-              disabled={btnDisabled}
-              icon={<ObjectIcon type={object.type} />}
-            >
-              {title}
-            </Button>
-          );
-          resultElements.push(element);
-        });
+        [...created, ...updated]
+          // .filter(object => supportsType(object.type))
```

Comment:
Guessing we'll enable this once all plugins are converted? Should put a TODO then

Reply:
Yep. Didn't want to break existing dashboard plugins and the more I think about it, dashboard plugins shouldn't have supported types. They're meant to be mounted in the react tree on any dashboard. If you had dashboard plugin supporting tables, you wouldn't be able to fetch the actual grid component to render the table as part of a panel with multiple components.